### PR TITLE
added

### PR DESCRIPTION
**Pull request by aastha01chauhan**
Please add hacktoberfest accepted label here

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -189,5 +189,15 @@
       "linkedin":"http://www.linkedin.com/",
       "quote":"Always Ready for It"
     }
+
+    {
+      "name": "AASTHA CHAUHAN",
+      "batch": 2024,IET LUCKNOW,
+      "picture_url": "https://ibb.co/P5XCzyC",
```
Comment (nlok5923) [inline data.json:196]:
```suggestion
      "picture_url": "https://i.ibb.co/khbc6Tc/IMG-9468.jpg",
```

File: data.json [hunk 2]
```diff
@@ -189,5 +189,15 @@
       "linkedin":"http://www.linkedin.com/",
       "quote":"Always Ready for It"
     }
+
+    {
+      "name": "AASTHA CHAUHAN",
+      "batch": 2024,IET LUCKNOW,
```
Comment (nlok5923) [inline data.json:195]:
```suggestion
      "batch": "2024 IET LUCKNOW",
```

**Review by nlok5923 — CHANGES_REQUESTED** [review]
PTAL

File: data.json [hunk 3]
```diff
@@ -189,5 +189,15 @@
       "linkedin":"http://www.linkedin.com/",
       "quote":"Always Ready for It"
     }
```
Comment (nlok5923) [inline data.json:191]:
```suggestion
    },
```

**Review by nlok5923 — APPROVED** [review]
Thanks !!

**Comment by aastha01chauhan** [comment]
made the changes, please check @nlok5923

**Comment by nlok5923** [comment]
> made the changes, please check @nlok5923

Approved your pr 🚀

**Pull request merged**